crates/evm/src/tests/queries/estimate_gas_tests.rs

```diff
@@ -110,7 +110,7 @@ fn test_tx_request_fields_gas() {
     );
     assert_eq!(
         contract_diff_size.unwrap(),
-        serde_json::from_value::<EstimatedDiffSize>(json![{"gas":"0x6601","l1DiffSize":"0x60"}])
+        serde_json::from_value::<EstimatedDiffSize>(json![{"gas":"0x6601","l1DiffSize":"0x1f"}])
```

Comment:
we need identical test that tests genesis diff size vs fork1 diff size

Reply:
adjusted all evm tests accordingly. there is already a test for it: `test_l1_fee_compression_discount`